tests/basic/tests/views.py

```diff
@@ -258,3 +258,8 @@ def test_private_cache_control(self):
 
         self.assertEqual(cache_control, set(["s-maxage=3600", "max-age=3600", "private"]))
         self.assertTrue('"johndoe"' in response.content.decode('utf-8'))
+
+    def test_filter_in(self):
+        response = self.client.get('/api/v1/notes/1/filterin/', HTTP_ACCEPT='application/json')
```

Comment:
In retrospect, I'm unhappy with this test.  It only shows that a view can call get_list with id__in a list, it doesn't actually show using the filter from the query parameters.